content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -223,19 +208,20 @@ This repair capability can be further configured with different RBAC permissions
 |`values.cni.repair.labelPods`    | UPDATE pods | Pods are only labeled.  User will need to take manual action to resolve.                                                                      |
 |`values.cni.repair.repairPods`   | None        | Pods are dynamically reconfigured to have appropriate configuration. When the container restarts, the pod will continue normal execution.     | Default in 1.21 and newer
 
-### Traffic redirection parameters
+### Traffic redirection parameters for sidecar dataplane mode
```

Comment:
We did say above that this guide was only for sidecar mode.  Is this reference content for the istio-cni component that should be on another page?

Reply:
Yes, this is a half-assed attempt to begin segregating the contents of this guide into "sidecar" and "not-sidecar" stuff, anticipating a better future split/shared doc.